client/my-sites/themes/theme-showcase.jsx

```diff
@@ -138,6 +138,10 @@ class ThemeShowcase extends Component {
 		this.props.setBackPath( this.constructUrl() );
 	}
 
+	isThemeDiscoveryEnabled = () =>
```

Comment:
We could add this to the constructor and save it as an instance variable instead of calculating it multiple times.

Reply:
It's not really an expensive computation. 🤔 
Also, I'm a bit rusty on class components, but I don't think we can be sure that `props.isLoggedIn` is accurate at construction time.